src/uct/rocm/copy/rocm_copy_iface.c

```diff
@@ -123,6 +125,8 @@ static uct_iface_ops_t uct_rocm_copy_iface_ops = {
     .iface_progress_enable    = ucs_empty_function,
     .iface_progress_disable   = ucs_empty_function,
     .iface_progress           = ucs_empty_function_return_zero,
+    .iface_event_fd_get       = (uct_iface_event_fd_get_func_t)ucs_empty_function_return_success,
+    .iface_event_arm          = (uct_iface_event_arm_func_t)ucs_empty_function_return_success,
```

Comment:
i think it's not valid to add UCT_IFACE_FLAG_EVENT_RECV, but not actually return any fd from iface_event_fd_get

Reply:
I can of course implement the proper solution here,  but I will wait on your feedback whether you see another approach for fixing this issue.